common/src/main/java/org/wso2/testgrid/common/ShellExecutor.java

```diff
@@ -88,7 +79,7 @@ public String getOutput() {
      * @return working directory
      */
     public String getWorkingDirectory() {
-        return workingDirectory.getAbsolutePath();
+        return workingDirectory.toString();
```

Comment:
This returns a relative path. Is that the intended use? Then, we need to add a javadoc explaining that.

Reply:
Updated the Javadoc comment.